javagen/src/main/java/com/azure/autorest/mapper/LiveTestsMapper.java

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.autorest.mapper;
+
+import com.azure.autorest.extension.base.model.codemodel.ScenarioStep;
+import com.azure.autorest.extension.base.model.codemodel.TestModel;
+import com.azure.autorest.extension.base.model.codemodel.TestScenarioStepType;
+import com.azure.autorest.model.clientmodel.ExampleLiveTestStep;
+import com.azure.autorest.model.clientmodel.LiveTestCase;
+import com.azure.autorest.model.clientmodel.LiveTestStep;
+import com.azure.autorest.model.clientmodel.LiveTests;
+import com.azure.autorest.model.clientmodel.ProxyMethodExample;
+import com.azure.autorest.util.CodeNamer;
+import com.azure.autorest.util.XmsExampleWrapper;
+import com.google.common.collect.Lists;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ * A mapper to map test model to live tests.
+ */
+public class LiveTestsMapper implements IMapper<TestModel, List<LiveTests>>{
```

Comment:
question: did you run this code?

Reply:
It's used here in `ClientMapper` to map scenario to vanilla `LiveTests`:
https://github.com/XiaofeiCao/autorest.java/blob/scenario_test_gen/javagen/src/main/java/com/azure/autorest/mapper/ClientMapper.java#L291

Reply:
I mean did you tried the code at least once, at runtime?

It is code in file and it compiles, but without a test-scenario in the input yaml, this part of code will never run?

Reply:
Yes it will run.  `FluentLiveTestsMapper` depends on `LiveTests` from `LiveTestMapper`, so it will run when generating `FluentLiveTests`:
https://github.com/Azure/autorest.java/blob/8b429b12ee9cb507ce1766b9dc91f795fb32edb3/fluentgen/src/main/java/com/azure/autorest/fluent/mapper/FluentLiveTestsMapper.java#L49

Reply:
I see. It is mapped twice for fluentgen?

Reply:
Yes it is...